Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/IfMultiplayerAgentType_ECBE9056-5EB3-AA98-AFB8-EC57E6C1BEA6.names

```diff
@@ -0,0 +1,51 @@
+{
+    "entries": [
+        {
+            "base": "{ECBE9056-5EB3-AA98-AFB8-EC57E6C1BEA6}",
+            "context": "ScriptCanvas::Node",
+            "variant": "",
+            "details": {
+                "name": "If Multiplayer Agent Type",
+                "category": "Multiplayer",
+                "tooltip": "An execution flow gate that continues to the path based on the agent type of the current multiplayer session."
+            },
+            "slots": [
+                {
+                    "base": "Input_In_0",
+                    "details": {
+                        "name": "In",
+                        "tooltip": "Branches on agent type"
```

Comment:
Do tooltips typically end with '.'? Not sure.

Reply:
Done.